docs/en/sql-reference/functions/conditional-functions.md

```diff
@@ -1,16 +1,16 @@
 ---
-description: 'Documentation for Conditional Functions'
+description: 'Documentation for conditional functions'
 sidebar_label: 'Conditional'
 sidebar_position: 40
 slug: /sql-reference/functions/conditional-functions
-title: 'Conditional Functions'
+title: 'Conditional functions'
 ---
 
-# Conditional Functions
+# Conditional functions
```

Comment:
Sure we no longer want uppercase first letters for all words of the title?
Asking because that would make it inconsistent with the titles of the other function documentation pages, for example [here](https://clickhouse.com/docs/sql-reference/functions/ym-dict-functions).